scipy/stats/tests/test_contingency.py

```diff
@@ -196,3 +196,28 @@ def test_chi2_contingency_bad_args():
     obs = np.empty((0, 8))
     assert_raises(ValueError, chi2_contingency, obs)
 
+
+def test_bad_association_args():
+    # Invalid Test Statistic
+    assert_raises(ValueError, association, [[1, 2], [3, 4]], "X")
+    # Invalid array shape
+    assert_raises(ValueError, association, [[[1, 2]], [[3, 4]]], "cramer")
+    # Invalid array shape (asymmetric)
+    assert_raises(ValueError, association, [[1, 2], [3, 4], [5]], 'cramer')
+    # chi2_contingency exception
+    assert_raises(Exception, association, [[-1, 10], [1, 2]], 'cramer')
+    # Invalid array (asymmetric)
+    assert_raises(ValueError, association, [[1, 2], [3]], 'cramer')
+    # Invalid array item value type
```

Comment:
This comment is hanging with not following code.  Did you have another test in mind?  If not, just delete the comment.

Reply:
add a test for an array with a string. must have missed it first time around.